src/Init/Prelude.lean

```diff
@@ -807,7 +807,7 @@ decidability instance instead of the proposition, which has no code).
 If a proposition `p` is `Decidable`, then `(by decide : p)` will prove it by
 evaluating the decidability instance to `isTrue h` and returning `h`.
 -/
-class inductive Decidable (p : Prop) where
+class inductive Decidable (p : Prop) : Type where
```

Comment:
This is a bug, it should not infer `Prop` here because the inductive has two constructors.

Reply:
Yes, the current draft makes a hacky tweak to `accLevel`, which I believe isn't the solution (as explained in the issue). This is not meant for review ! When tweaking `mkResultUniverse`, I end up getting some kernel errors regarding loose universe variables in sort-polymorphic inductives like `PProd`, so there's still work to do.